test/unit/test_diff_getter.py

```diff
@@ -0,0 +1,317 @@
+"""
+test_diff_getter.py
+Date:  23/07/2020
+Author: Eugeniu Costetchi
+Email: costezki.eugen@gmail.com 
+"""
+from unittest.mock import patch
+from collections import namedtuple
+
+import pytest
+import requests
+from SPARQLWrapper import SPARQLWrapper
+
+from rdf_differ.diff_getter import FusekiDiffGetter
+
+RequestObj = namedtuple('RequestObj', ['status_code', 'url', 'text'])
+
+
+def test_FusekiDiffGetter_sparql_endpoint():
+    fuseki_service = FusekiDiffGetter(triplestore_service_url="http://localhost:3030")
+    assert fuseki_service.make_sparql_endpoint(dataset_name="subdiv") == "http://localhost:3030/subdiv/sparql"
+    assert fuseki_service.make_sparql_endpoint(dataset_name="/foe") == "http://localhost:3030/foe/sparql"
+
+    fuseki_service = FusekiDiffGetter(triplestore_service_url="http://localhost:3030/")
+    assert fuseki_service.make_sparql_endpoint(dataset_name="subdiv") == "http://localhost:3030/subdiv/sparql"
+    assert fuseki_service.make_sparql_endpoint(dataset_name="/foe") == "http://localhost:3030/foe/sparql"
+
+
+@patch.object(requests, 'get')
```

Comment:
I'm not a big fan of patching tbh. The reason is that by patching, you are coupling the tests with the implementation. And also, there is a saying "Don't Mock What You Don't Own".